PUBLISH.md

```diff
@@ -1,33 +1,56 @@
-# Releasing a new version
+# Version management
 
-To be able to do `docker push` you need to have your Docker Hub account added to the dedis organisation.
-Here are the steps we use to make a conode binary release:
-```
-git checkout master
-git diff -> be sure you have no outstanding changes
-git log
-```
-review to be sure this is where the tag should be
+For the latest version of the conodes, please use 
+https://github.com/c4dt/byzcoin, which provides a continuous release of valid
+ conodes.
```

Comment:
Commenting here even though that belongs to `https://github.com/c4dt/byzcoin`:

For the version string associated to the ByzCoin release, I would suggest to have, in addition to the tag and date, the git commit. This would allow to pinpoint precisely to the code that is running. Something similar to what `git describe` produces:
```$ git describe --tags
v3.4.5-85-g3ce8cc8a6
```
What do you think?

Reply:
Sounds good - can you please open an issue at c4dt/byzcoin? You can assign it to me. There might be some more discussion ;)

Reply:
[Done](c4dt/byzcoin#28)